toolbox/repo/validate_role_vars_used.py

```diff
@@ -0,0 +1,78 @@
+#! /usr/bin/env python
+
+import glob, os
+import yaml
+import subprocess
+import pathlib
+
+THIS_DIR = pathlib.Path(__file__).absolute().parent
+
+# go to top directory
+os.chdir(os.path.join(THIS_DIR, "..", ".."))
+
+ROLES_VARS_GLOB = "roles/*/vars/*/*"
+ROLES_DEFAULTS_GLOB = "roles/*/defaults/*/*"
+
+def validate_role_vars_used(filename, yaml_doc):
+    errors = 0
+
+    role_name = filename.split("/", 2)[1]
```

Comment:
Does the `, 2` in the split make any difference in this case?

Reply:
Also, you can use `pathlib.Path(filename).parts` to get a list of path components you can index into

Reply:
using `Path` everywhere, it now gives `filename.parts[len(TOP_DIR.parts)+1]` :)